lib/matplotlib/contour.py

```diff
@@ -702,6 +702,11 @@ def __init__(self, ax, *args,
         self._extend_min = self.extend in ['min', 'both']
         self._extend_max = self.extend in ['max', 'both']
         if self.colors is not None:
+            if mcolors.is_color_like(self.colors):
+                color_sequence = [self.colors]
```

Comment:
👍 for using a new name and not modifying `colors`. This makes it clear it's a local name and prevents users from thinking its the argument `colors`, which also gets stored as `self.colors`.